From khali@linux-fr.org Wed Oct 26 12:25:25 2005
Date: Wed, 26 Oct 2005 21:21:50 +0200
From: Jean Delvare <khali@linux-fr.org>
To: Greg KH <greg@kroah.com>
Subject: [PATCH 08/16] i2c: i2c-i810 documentation update
Message-Id: <20051026212150.010e7472.khali@linux-fr.org>
Content-Disposition: inline; filename=i2c-i810-update-device-list.patch

Update the documented list of devices supported by the i2c-i810
driver.

Signed-off-by: Jean Delvare <khali@linux-fr.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 Documentation/i2c/busses/i2c-i810 |    1 +
 drivers/i2c/busses/Kconfig        |    3 ++-
 drivers/i2c/busses/i2c-i810.c     |    1 +
 3 files changed, 4 insertions(+), 1 deletion(-)

--- gregkh-2.6.orig/Documentation/i2c/busses/i2c-i810
+++ gregkh-2.6/Documentation/i2c/busses/i2c-i810
@@ -2,6 +2,7 @@ Kernel driver i2c-i810
 
 Supported adapters:
   * Intel 82810, 82810-DC100, 82810E, and 82815 (GMCH)
+  * Intel 82845G (GMCH)
 
 Authors: 
 	Frodo Looijaard <frodol@dds.nl>, 
--- gregkh-2.6.orig/drivers/i2c/busses/Kconfig
+++ gregkh-2.6/drivers/i2c/busses/Kconfig
@@ -135,11 +135,12 @@ config I2C_I810
 	help
 	  If you say yes to this option, support will be included for the Intel
 	  810/815 family of mainboard I2C interfaces.  Specifically, the 
-	  following versions of the chipset is supported:
+	  following versions of the chipset are supported:
 	    i810AA
 	    i810AB
 	    i810E
 	    i815
+	    i845G
 
 	  This driver can also be built as a module.  If so, the module
 	  will be called i2c-i810.
--- gregkh-2.6.orig/drivers/i2c/busses/i2c-i810.c
+++ gregkh-2.6/drivers/i2c/busses/i2c-i810.c
@@ -32,6 +32,7 @@
    i810AB		7123           
    i810E		7125           
    i815			1132           
+   i845G		2562
 */
 
 #include <linux/kernel.h>
